core/tests/protractor_desktop/editorFeatures.js

```diff
@@ -973,6 +973,51 @@ describe('Exploration translation', function() {
     explorationEditorTranslationTab.changeTranslationLanguage('Hindi');
   });
 
+  it('the translation subtab should not change on saving draft', function() {
```

Comment:
components tab* (Also, start with `should` such that the describe(string) + it(string) will make a complete sentence. see other parts of tests for reference.)

Reply:
Would `'should maintain its active sub-tab on saving draft and publishing changes'` be fine?